cmd/ocm-support/utils/utils.go

```diff
@@ -41,4 +41,5 @@ var Aliases = map[string][]string{
 	"capability":   {"cap"},
 	"organization": {"org"},
 	"subscription": {"sub"},
+	"account":      {"acc"},
```

Comment:
Let's consolidate the aliases used by ocm-cli. They use acct(s).

Reply:
changed